content/agent/proxy.md

```diff
@@ -31,8 +31,8 @@ Traditional web proxies are supported natively by the Agent. If you need to conn
 
 ### Agent v6
 
-With Agent v6 you can set different proxy servers for `https` and `http` requests in your `datadog.yaml` file. The Agent always uses `https` to send data to Datadog, but integrations might use `http` to collect metrics from different software.
-No matter the proxied queries, you can activate SSL or not on your proxy server. Find below some configuration examples for your `datadog.yaml` file:
+With Agent v6, you can set different proxy servers for `https` and `http` requests in `datadog.yaml`. The Agent uses `https` to send data to Datadog, but integrations might use `http`.
```

Comment:
@olivielpeau @hush-hush can you confirm the language here? I dont think setting proxies will guarantee that integrations will use the proxy. Just that agent to datadog communications will?

Reply:
I ask integrations-core to join the review on this.

Reply:
@DataDog/agent-integrations can you help review? We should get this out ASAP since 6.4.x has now been released.